src/main/java/org/springframework/data/elasticsearch/core/query/NativeSearchQuery.java

```diff
@@ -111,6 +113,15 @@ public void addScriptField(ScriptField... scriptField) {
         scriptFields.addAll(Arrays.asList(scriptField));
     }
 
+	@Override
+	public CollapseBuilder getCollapse() {
```

Comment:
```suggestion
	public CollapseBuilder getCollapseBuilder() {
```
the field is `collapseBuilder` the type is a Builder, so I think this method name should reflect this